### DISCUSSION TITLE
HA via VM auf Synology hmip-rfusb Integration funktioniert nicht

### DISCUSSION THREAD

**Discussion by alphabravocharly72** [SukramJ/aiohomematic#1080] [General]
Hallo,
soweit läuft meine Raspberry Matic auf HA (VM auf Synology). Nur die Integration mit HA funktioniert nicht.
Homematic(IP) Local funktioniert mit der CCU3 aber nicht mit dem Stick
Der hmip-rfusb Stick wird auch nicht in der HA Hardware angezeigt. Liegt es an dem? 
Steige von meiner CCU3 auf den USB Stick um. 
Danke für eure Hilfe
alpha

**Comment by alphabravocharly72** [comment]
Hab alle Addons in HA gelöscht und alles neu installiert jetzt hat HA Zugriff auf den USB Stick.
Habe das backup der CCU3 eingespielt, läuft azch in der  RaspberryMatic, nur sehe ich meine Geräte in HA nicht.
Kann es sein das ich die Geräte neu anlernen muss um auch inHA sichtbar zu sein?
Danke für eure Hilfe
alpha

**Comment by alphabravocharly72** [comment]
Hab jetzt ein Gerät neu angelernt. Wird nicht in HA angezeigt. Hat wer eine Idee?
Danke
alpha

**Reply by danielperna84** [reply]
Aber im RaspberryMatic ist das Gerät zu sehen und auch steuerbar? Wenn ja, dann sollte das Problem eigentlich nur an der Kommunikation zwischen der Integration und dem Add-on scheitern. Hier bitte peinlich genau der Anleitung aus dem RaspberryMatic Repo folgen. Und die Einschränkungen bzgl. Sonderzeichen im Passwort vom API User berücksichtigen.

**Comment by alphabravocharly72** [comment]
ja das Gerät ist zu sehen und steuerbar. PW API User ist nicht gesetzt. Es sind auch keine Sonderzeichen der Namen vergeben. Es funktioniert weder das Backup der CCU3 noch ein neu angelerntes Gerät. In RaspberryMatic (HA) läuft alles.  Hier die Geräte die ich bekomme. mehr leider nicht.
<img width="251" alt="image" src="https://github.com/danielperna84/hahomematic/assets/59312982/c5374ed0-9ff0-4f9d-bcfd-e6723c7bf11a">

**Reply by danielperna84** [reply]
> PW API User ist nicht gesetzt.

Und du hast [diesen](https://github.com/danielperna84/custom_homematic#authentication) Absatz auch gelesen und verstanden?

**Comment by alphabravocharly72** [comment]
das Häcken ist gesetzt. Bin auch als Admin eingeloggt.
<img width="549" alt="image" src="https://github.com/danielperna84/hahomematic/assets/59312982/97c45819-3c2d-4f6e-ae6c-d0a093e370ed">

Muss ich hier was einstellen?
<img width="275" alt="image" src="https://github.com/danielperna84/hahomematic/assets/59312982/79e5e84f-1f1f-4347-888b-0168b1b73f45">


Danke für deine Hilfe
alpha

**Reply by danielperna84** [reply]
Kann man normalerweise so lassen. Muss man eigentlich nur anpassen, wenn man sowas wie Docker benutzt, oder eine Firewall dazwischen ist. TLS musst du nur aktivieren, wenn du RaspberryMatic entsprechend konfiguriert hast. Im Default ist das so also schon alles richtig.

**Comment by alphabravocharly72** [comment]
komisch warum es dann nicht funktioniert

**Reply by danielperna84** [reply]
Logs könnten hilfreich sein.

**Comment by alphabravocharly72** [comment]
Jetzt funktioniert es. Habe das Add on nochmals neu installiert. 
Danke für deine Hilfe.
alpha
